pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -242,22 +245,29 @@ def brew_login(self):
     async def regen_repo_with_complete(self, el_v):
         build_tag = f'rhaos-{self.ocp_version}-rhel-{el_v}-build'
         self.brew_login()
+        if self.dry_run:
+            _LOGGER.info(f"[DRY RUN] Would have regenerated repo for tag: {build_tag}")
+            return False
         task_id = self.koji_session.newRepo(build_tag)
         _LOGGER.info(f"Regenerating repo for tag: {build_tag} with task {task_id}, waiting...")
         return await watch_task_async(self.koji_session, _LOGGER.info, task_id)
 
-    def tag_builds(self, el_v, nvr):
+    def tag_build(self, el_v, nvr):
         build_tag = f'rhaos-{self.ocp_version}-rhel-{el_v}-override'
         self.brew_login()
+        nvrs_to_tag = {nvr}
```

Comment:
the nvr parameter should be one nvr for rhel9. rhel8 is special due to 3 module builds